performancetest/src/main/java/org/corfudb/performancetest/LogUnitPerformanceTest.java

```diff
@@ -0,0 +1,138 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.LogUnitClient;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * Created by Nan Zhang and Lin Dong on 10/23/19.
+ */
+
+public class LogUnitPerformanceTest {
+    private String endPoint = "localhost:9000";
+    private int metricsPort = 1000;
+    private long seed = 1024;
+    private int randomBoundary = 100;
+    private Random random;
+    static MetricRegistry metricRegistry;
+    Timer readTimer;
+    Timer writeTimer;
+
+    private static final String METRIC_PREFIX = "corfu-perf";
+    private static final Properties PROPERTIES = new Properties();
+    private static final String READ_PERCENT = "logunitReadPercent";
+    private static final String SINGLE_REQUEST = "logunitSingleRequests";
+    private static final String BATCH_REQUEST = "logunitBatchRequests";
+    private static final String BATCH_SIZE = "logunitBatchSize";
+    private static final String ENTRY_SIZE = "logEntrySize";
+
+    public LogUnitPerformanceTest() {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        InputStream input = classLoader.getResourceAsStream("PerformanceTest.properties");
+
+        try {
+            PROPERTIES.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        random = new Random(seed);
+        metricRegistry = CorfuRuntime.getDefaultMetrics();
+        readTimer = metricRegistry.timer(METRIC_PREFIX + "logunit-single-read");
+        writeTimer = metricRegistry.timer(METRIC_PREFIX + "logunit-single-write");
```

Comment:
find a unified way to report result, all by metrics or logs

Reply:
Can you explain more?

Reply:
You can use metrics reporter to write files, or you can print logs into files.
I mean you should select one way and remove another one.